src/wizard/index.tsx

```diff
@@ -90,6 +100,11 @@ export default function Wizard({
 
   useEffect(() => {
     trackStartWizard();
+
+    return () => {
+      trackEndWizard(funnelId);
```

Comment:
Why does "start" not receive an id but "end" does?

Reply:
Maybe trackStartWizard is incorrectly named as it is not sending a metric and only creates a timestamp for component mount and then trackStep will send the metric. Since there is no trackEndStep, trackEndWizard will create an end timestamp as well as send a metric.

Reply:
This aligns with my another comment: https://github.com/cloudscape-design/components/pull/745#discussion_r1112894071

Let's keep the current state for now